src/ui/Auth.tsx

```diff
@@ -1,22 +1,23 @@
 import { authenticateUser } from '../data'
 import googleLogo from '../icons/google-logo.png'
 import githubLogo from '../icons/github-logo.png'
+import { Provider } from '@supabase/supabase-js'
 
 const providers = [
   {
-    provider: 'google',
+    provider: 'google' as Provider,
```

Comment:
Please try to avoid using as, it basically turns off typescript when we do that.